README.md

```diff
@@ -121,8 +128,45 @@ PickleSerializer:
 SESSION_SERIALIZER = 'django.contrib.sessions.serializers.PickleSerializer'
 ```
 
+## Configuration
+
+### Django session login
+
+Django session login is the usual login to Django that sets up a session
+and is typically implemented using a browser cookie. After authentication
+is completed everything proceeds, as if the user had logged in using
+username and password in the login form. For us, the main use case has been
+allowing logins to Django admin.
+
+Each installation ("client" in OIDC parlance) will need its own configuration in Tunnistamo and
```

Comment:
Perhaps add a header here. IMO the subject changes a bit. Maybe something like `### Using Tunnistamo`.

Reply:
It should be the same subject, I'll rewrite this.

Reply:
@akikoskinen The subject actually remains the same, as far as I understand it. This configuration is only needed if the Django UI needs Tunnistamo integration. Right, @vikoivun  ?

Reply:
Right. But on re-reading it was also obvious why it looked like a sudden subject switch.